tests/unit/test_ai_loop.py

```
@@ -307,6 +307,82 @@ def test_stop_yields_ai_stopped(self):
+
+		def _fake_dispatch_action(action, c):
+			yield follow_up
+
+		with patch("secator.tasks.ai.safe_dispatch_action", _fake_dispatch_action):
+			gen = AiTask._dispatch_and_collect(fake_self, [{"tool_call_id": "tc_fu"}], ctx)
+			yielded = list(gen)
+		return yielded, persisted, follow_up
+
+	def test_remote_follow_up_persisted_pending_with_choices(self):
+		backend = RemoteBackend(timeout=60, query_engine=MagicMock())
+		yielded, persisted, follow_up = self._run_dispatch(backend)
```

Ruff (0.15.18)

[warning] Unpacked variable `yielded` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)